kaali.js

```diff
@@ -0,0 +1,1371 @@
+  });
+});
+//load alert.html for mobile
+$(window).load(function () {
+  $(function () {
+    $("#component-alert").click(function () {
+      displayProcedure();
+      removeActiveClass();
+      document.querySelector("#alertlink").classList.add("active");
+      $("#main").load("./components/alert/alert.html");
+    });
+  });
+});
+// ---------------------------------------------------------------------------------------------
+
+// Common Procedure to load components on mobile
+function displayProcedure() {
+  document.querySelector(".navbar-ul").style.display = "none";
+  document.querySelector(".close-menu").style.display = "none";
+  document.querySelector(".fas.fa-bars").style.display = "block";
+}
+// ---------------------------------------------------------------------------------------------
+// Making navbar responsive STEP 2
+$(window).resize((e) => {
+  if (parseInt(window.innerWidth) <= 985) {
+    document.querySelector(".navbar-ul").style.display = "none";
+    document.querySelector(".close-menu").style.display = "none";
+    document.querySelector(".fa-bars").style.display = "block";
+  } else {
+    document.querySelector(".navbar-ul").style.display = "flex";
+    document.querySelector(".fa-bars").style.display = "none";
+    document.querySelector(".close-menu").style.display = "none";
+  }
+
+  document.querySelector(".fa-bars").addEventListener("click", () => {
+    if (document.querySelector(".navbar-ul").style.display === "none") {
+      document.querySelector(".navbar-ul").style.display = "block";
+      document.querySelector(".navbar-ul").classList.add("slide-in-top");
+
+      // Hide hamburger
+      document.querySelector(".fa-bars").style.display = "none";
+
+      // Show Close Button
+      document.querySelector(".close-menu").style.display = "block";
+    }
+  });
+
+  document.querySelector(".close-menu").addEventListener("click", () => {
+    if (document.querySelector(".close-menu").style.display == "none") {
+      // document.querySelector(".close-menu").style.display = "block";
+    } else {
+      document.querySelector(".navbar-ul").classList.add("slide-out-top");
+      document.querySelector(".close-menu").style.display = "none";
+      document.querySelector(".fa-bars").style.display = "block";
+
+      document.querySelector(".navbar-ul").style.display = "none";
+    }
+  });
+});
+
+// remove active class
+function removeActiveClass() {
+  let activeclass = document.querySelector(".active");
+
+  if (activeclass != null) {
+    activeclass.classList.remove("active");
+  }
+}
+// ---------------------------------------------------------------------------------------------
+
+// Avatar
+
+//load avatar.html for mobile
+$(window).load(function () {
+  $(function () {
+    $("#component-avatar").click(function () {
+      displayProcedure();
+      removeActiveClass();
+      document.querySelector("#avatarlink").classList.add("active");
+      $("#main").load("./components/avatar/avatar.html");
```

Comment:
Should create a reusable function for this which accepts path and id as param.

Reply:
removed that function itself